neard/src/cli.rs

```diff
@@ -70,7 +69,7 @@ impl NeardCmd {
             NeardSubCommand::Run(cmd) => cmd.run(&home_dir, genesis_validation, runtime),
 
             NeardSubCommand::StateViewer(cmd) => {
-                let mode = if cmd.readwrite { Mode::ReadWrite } else { Mode::ReadOnly };
+                let mode = near_store::Mode::read_write(cmd.readwrite);
```

Comment:
As mentioned in the above comment, I would rather keep the existing code and not introduce the constructor.